authorization/config.yaml

```diff
@@ -2,14 +2,14 @@ log:
   debug: false
 apis:
   grpc:
-    address: "0.0.0.0:9000"
+    address: "0.0.0.0:9100"
```

Comment:
why are you changing ports in the config refactoring branch?

Reply:
Default port was 9100